### PR TITLE
feat: more info for the select from application screen

### PR DESCRIPTION
**Pull request by Tyffhabwe**

### CODE REVIEW

File: app/src/main/java/app/revanced/manager/ui/screen/AppSelectorScreen.kt [hunk 1]
```diff
@@ -85,28 +89,62 @@ fun AppSelectorScreen(
...
+                        Icon(
+                            imageVector = Icons.Outlined.Search,
+                            contentDescription = stringResource(R.string.search),
+                            modifier = Modifier.size(64.dp)
+                        )
+
+                        Text(
+                            text = stringResource(R.string.type_anything),
+                            style = MaterialTheme.typography.bodyLarge
+                        )
```
Comment (Axelen123) [inline app/src/main/java/app/revanced/manager/ui/screen/AppSelectorScreen.kt:144]: What is this for?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -92,6 +92,8 @@
     <string name="ok">OK</string>
     <string name="patch">Patch</string>
     <string name="select_from_storage">Select from storage</string>
+    <string name="select_an_installation_file">Select an installation file using file picker</string>
```
Comment (Axelen123) [inline app/src/main/res/values/strings.xml:95]: Can you rename this to `select_from_storage_description` to keep it consistent?

**Review by Axelen123 — CHANGES_REQUESTED** [review]

File: app/src/main/java/app/revanced/manager/ui/screen/AppSelectorScreen.kt (hunk 1 above)
Comment (Tyffhabwe) [reply app/src/main/java/app/revanced/manager/ui/screen/AppSelectorScreen.kt:144]: App selector screen
image

**Comment by Tyffhabwe** [comment]
Screenshot_2023-08-22-21-34-11-03_a7c8ff5d38679aafe3a21eaf88d67b8c
Screenshot_2023-08-22-21-34-17-51_a7c8ff5d38679aafe3a21eaf88d67b8c

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -92,6 +92,8 @@
     <string name="ok">OK</string>
     <string name="patch">Patch</string>
     <string name="select_from_storage">Select from storage</string>
+    <string name="select_from_storage_description">Select an installation file using file picker</string>
```
Comment (oSumAtrIX) [inline app/src/main/res/values/strings.xml:95]:
```suggestion
    <string name="select_from_storage_description">Select an APK file from storage using file picker</string>
```

**Review by oSumAtrIX — COMMENTED** [review]
A lot of formatting changes have been made. Please make sure only to make necessary changes next time and formatting changes separately to simplify the review.

**Review by Axelen123 — APPROVED** [review]

**Pull request merged**